README.md

```diff
@@ -4,11 +4,11 @@
 - The sample project contains 42 samples, and it still grows.    
 - These samples are grouped by Azure services and [Spring Cloud Azure libraries](https://github.com/Azure/azure-sdk-for-java/tree/feature/azure-spring-cloud-4.0/sdk/spring).    
 
-    > For example: [single-namespaces](servicebus/spring-cloud-azure-starter-integration-servicebus/single-namespaces), it located in the path `/servicebus/spring-cloud-azure-starter-integration-servicebus`.
+    > For example: [single-namespace](servicebus/spring-cloud-azure-starter-integration-servicebus/single-namespace), it located in the path `/servicebus/spring-cloud-azure-starter-integration-servicebus`.
```

Comment:
I was suggesting we change it to `[Use Spring Integration with single Azure Service Bus namespace](servicebus/spring-cloud-azure-starter-integration-servicebus/single-namespace)`